src/components/Box/Box.js

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { GameContext } from '../../context/GameContext.js';
+import './Box.css';
+
+export default function Box({ box, index }) {
+  // const { active, board, setBoard, currentPlayer, setCurrentPlayer, setGameMessage } =
+  //   useContext(GameContext);
+
+  const { handleChoice } = useContext(GameContext);
+
+  // function handleChoice() {
```

Comment:
Try to remember to delete commented out code, unless you plan on using it soon, its best practice to not have it in your commits.

Reply:
I will make a note of this, and remember to clean things up before next submission, thanks Bradley!